tools/issue-labeler/src/SearchIndexCreator/Program.cs

```diff
@@ -0,0 +1,219 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.Extensions.Configuration;
+using Octokit;
+using Azure.Identity;
+using Azure.AI.OpenAI;
+using Azure.Search.Documents.Indexes;
+using AzureRAGService;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+
+namespace SearchIndexCreator
```

Comment:
This class should just direct traffic.   Provide the menu and then reach out to the right class to do whatever command was selected.

Each command should be its own class so that we can segment logic and keep logical operations organized and separated.